src/otx/recipe/_base_/data/mmaction_base.yaml

```diff
@@ -78,6 +85,8 @@ config:
         scale:
           - -1
           - 256
+      - type: CenterCrop
```

Comment:
Is that center crop reproduced in OV action model?

Reply:
Nope, but it is for sync with previous OTX1.x pipeline. I'll revisit this pipeline after fixing some performance issue in action classification. Currently there's accuracy drop in OTX2.0